crypto/zk/circuit/config.go

```diff
@@ -92,17 +92,16 @@ var CircuitsConfigurations = map[string]*ZkCircuitConfig{
 		WasmHash:                hexToBytes("0x80a73567f6a4655d4332301efcff4bc5711bb48176d1c71fdb1e48df222ac139"),
 		WasmFilename:            "circuit.wasm",
 	},
-	"prod": {
-		URI: "https://raw.githubusercontent.com/vocdoni/" +
-			"zk-franchise-proof-circuit/master",
-		CircuitPath:             "artifacts/zkCensus/dev/160",
+	"v1.0.0": {
+		URI:                     "https://raw.githubusercontent.com/altergui/zk-voceremony",
```

Comment:
:warning: i published the ceremony result in my fork. don't merge until a formal release is published.

Reply:
i did publish this in https://github.com/vocdoni/zk-voceremony/pull/139